### ISSUE TITLE
Deformed keratohyaline granules

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9808]
**Preferred term label:**
Deformed keratohyaline granules

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/618084 "Electron microscopy revealed intraepidermal separation within the lower corneal layers and a reduction in the number of keratohyalin granules, most of which were deformed and irregularly shaped"

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
Irregularly shaped keratohyaline granules
HP:0034944
